trlc/trlc.py

```diff
@@ -40,6 +40,9 @@
 except ImportError:  # pragma: no cover
     VCG_API_AVAILABLE = False
 
+# Ignore SIGPIPE (Broken pipe) signal
+signal.signal(signal.SIGPIPE, signal.SIG_DFL)
```

Comment:
Please try `SIG_IGN` and check what happens. Does the program exist with return code 141 as with `SIG_DFL`, or with some other value?